defaults/main.yml

```diff
@@ -1,9 +1,6 @@
 ---
-
 letsencrypt_install_dir: /opt/letsencrypt/
 letsencrypt_cert_dir: /etc/ssl/letsencrypt/
 letsencrypt_webroot_dir: /var/www/acme-challenges/
 letsencrypt_skip_selftest: False
 letsencrypt_email: contact@example.com
-letsencrypt_cert_domains:
-  example.com: [www.example.com, foo.example.com]
```

Comment:
I'm a friend of have all relevant variables one might want to overwrite in the defaults/main.yml. I see why you wouldn't want to have this enabled though. Would it be acceptable to keep, but comment out letsencrypt_email and letsencrypt_cert_domains?